src/components/GroupedList/GroupedList.Props.ts

```diff
@@ -167,6 +167,9 @@ export interface IGroupHeaderProps {
 
   /** Callback for when the group is selected. */
   onToggleSelectGroup?: (group: IGroup) => void;
+
+  /** Determines if the group selection check box is shown for collapsed groups. */
+  preventSelectCollapsedGroups?: boolean;
```

Comment:
This IS a header concern, so I added it in IGroupHeaderProps... let's look at this together and clean up the interfaces together..